test/snippet/io/sequence_file/sequence_file_output_fields_trait_1.cpp

```diff
@@ -7,26 +7,34 @@
 #include <seqan3/alphabet/quality/phred42.hpp>
 #include <seqan3/alphabet/quality/qualified.hpp>
 #include <seqan3/io/sequence_file/output.hpp>
+#include <seqan3/range/views/get.hpp>
 
 int main()
 {
     using seqan3::operator""_dna5;
     using seqan3::operator""_phred42;
 
     seqan3::sequence_file_output fout{std::ostringstream{},
-                                      seqan3::format_fasta{},
-                                      seqan3::fields<seqan3::field::id, seqan3::field::seq_qual>{}};
+                                      seqan3::format_fastq{},
+                                      seqan3::fields<seqan3::field::id, seqan3::field::seq, seqan3::field::qual>{}};
 
     for (int i = 0; i < 5; i++)
     {
         std::string id{"test_id"};
         // vector of combined data structure:
-        std::vector<seqan3::qualified<seqan3::dna5, seqan3::phred42>> seq_qual{{'N'_dna5, '7'_phred42}};
+        std::vector<seqan3::qualified<seqan3::dna5, seqan3::phred42>> seq_qual{{'N'_dna5, '7'_phred42},
+                                                                               {'A'_dna5, '1'_phred42},
+                                                                               {'C'_dna5, '3'_phred42}};
+
+        auto view_on_seq  = seqan3::views::get<0>(seq_qual);
+        auto view_on_qual = seqan3::views::get<1>(seq_qual);
 
         // ...
 
-        fout.emplace_back(id, seq_qual);       // note also that the order the argumets is now different, because
-        // or:                                    you specified that ID should be first in the fields template argument
-        fout.push_back(std::tie(id, seq_qual));
+        fout.emplace_back(id, view_on_seq, view_on_qual); // Also note that the order of the arguments is now
+                                                          // different, because you specified that ID should be
+                                                          // first in the fields template argument
```

Comment:
I analysed the comment too long:


Maybe something along the lines of:

```
Note that the order of the arguments is different from the default `seq, id, qual`, because you specified that ID should be first in the fields template argument.
```

* I removed the `Also` because this is the first thing to note.
* I don't like the use of `now` in the sentence because there is no point of reference as to what it was before.
* For the same reason, I added the default order.
* I also added a period at the end :)
* I think this comment could go to the line above `fout.emplace_back` and use the full length...